scss/_navbar.scss

```diff
@@ -176,7 +176,7 @@
         }
 
         .navbar-collapse {
-          display: flex !important;  // stylelint-disable-line declaration-no-important
+          display: flex !important; // stylelint-disable-line declaration-no-important
```

Comment:
This improves nothing; it depends on your editor. I don't mind having though.

Reply:
We need that comment for stylelint, `npm run css-lint` sequence.
It comes from #23572, [here](https://github.com/twbs/bootstrap/pull/23572/files#diff-df1f74faf207a474f1d78c86393221c8R178) :)